src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -90,6 +90,23 @@
   margin-top: 20px;
 }
 
+.descriptionWrapper :first-child {
+  margin: 0;
+}
```

Comment:
I don't think this does anything, since our CSS normalization stylesheet resets most basic elements' margins to 0. Could you remove this?